src/raftstore/store/store.rs

```diff
@@ -335,7 +335,7 @@ impl<T: Transport, C: PdClient> Store<T, C> {
                         let mut p = self.region_peers.remove(&region_id).unwrap();
                         let end_key = enc_end_key(&p.region());
                         if let Err(e) = p.destroy() {
-                            error!("destroy peer {:?} for region {} in store {} err {:?}",
+                            panic!("destroy peer {:?} for region {} in store {} err {:?}",
```

Comment:
If user just restart the program to try to recover from failure, this may cause problem.